source/ed/connectors/gtfs_parser.cpp

```diff
@@ -1257,6 +1258,9 @@ void GenericGtfsParser::fill_default_modes(Data& data) {
 void normalize_extcodes(Data& data) {
     for (nm::StopArea* sa : data.stop_areas) {
         boost::algorithm::replace_first(sa->uri, "StopArea:", "");
+        if (sa->from_orphan_stoppoint) {
+            sa->uri = "Navitia:" + sa->uri;
+        }
```

Comment:
Why not doing a straight ` sa->uri = "Navitia:" + sp->uri;` when creating the StopArea in `StopsGtfsHandler::handle_stop_point_without_area` ? This way, you wouldn't need `from_orphan_stoppoint` variable.

Reply:
this is to simplify the links and retrieve the properties of the stoppoint from files:
comments.txt
equipments.txt
object_properties.txt
object_codes.txt
admin_stations.txt